packages/contracts/contracts/engine/V3/forks/PROHIBITION/GenArt721CoreV3_Engine_Flex_PROHIBITION.sol

```diff
@@ -1124,6 +1127,14 @@ contract GenArt721CoreV3_Engine_Flex_PROHIBITION is
         _onlyNonEmptyString(_projectName);
         _onlyNonZeroAddress(_artistAddress);
         require(!newProjectsForbidden, "New projects forbidden");
+        require(
+            adminACLAllowed(
+                msg.sender,
+                address(this),
+                this.addProject.selector
+            ),
+            "Only AdminACL allowed"
+        );
```

Comment:
Just made this change and it caused the whole contract to go over the size limit :/

Reply:
oops! taking a look now...

Reply:
Figured something out to get around it

Reply:
@tlip alright - should be okay to replace this with this with the following:
```
        require(
            owner() == address(0) ||
                adminACLAllowed(
                    msg.sender,
                    address(this),
                    this.addProject.selector
                ),
            "Only Admin ACL allowed"
        );
```
It works because:
- short-circuits and allows projects to be added if owner has renounced ownership
- (slightly) updates revert string to be same as a different revert string elsewhere, and solidity compiler seems to be smart enough to re-use that chunk of bytecode and stay under smart contract size limit

This assumes the owner doesn't renounce ownership prior to reaching project 10, which seems reasonable 😄

Reply:
*(was out of date and missed your previous message, glad you found something that worked!)

Reply:
ngl i was sitting here contemplating which the better option is hahah

Reply:
resolving based on latest commits